tests/foreman/cli/test_gpgkey.py

```diff
@@ -1183,7 +1183,7 @@ def test_positive_consume_content_using_repo(self):
 
         :CaseAutomation: notautomated
 
-        :CaseLevel: System
+        :CaseLevel: Integration
```

Comment:
If the `:CaseLevel: Integration` and `::CaseLevel: Component` then would make more sense to change the tier3 to tier2.

Reply:
done